services/web-app/utils/sendLocalRequest.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { getRunMode, RunMode } from '@carbon-platform/api/runtime'
+import cookie from 'cookie'
+
+/**
+ * Retrieves and returns serialized auth cookie
+ *
+ * @param {NextRequest} req getServerSideProps request object
+ * @returns {string} serialized auth cookie
+ */
+function getAuthCookie(req) {
+  const sessionCookie = req.cookies['connect.sid']
+  return sessionCookie ? cookie.serialize('connect.sid', sessionCookie) : ''
+}
+
+/**
+ * Sends an http request internally
+ * @param {NextRequest} req - Original request object that came through the middleware
+ * or getServerSideProps.context
+ * @param {string} path - Relative path to request
+ * @param {boolean} forwardAuthCookie - Should auth cookie be forwarded to local request
+ * @param {string} method - Request method to use, default to 'GET'
+ * @param {Object} body - Body object to send to fetch command
+ * @param {Object} requestOptions - Additional request options to send to the fetch command
+ * @returns {Promise<Response>} Promise return from fetch()
+ */
+export default function sendLocalRequest(
```

Comment:
There's a fetch in `retrieveUser.js` that could benefit from this function. We should definitely update that one either in this PR or in a separate one.

There's also a couple in `auth.js`, but those are so simple it might not be worth it.

Reply:
Now using in `retrieveUser.js`, `auth.js` doesn't have the absolute url constraint so I don't think it's necessary